src/components/common/navbar.tsx

```diff
@@ -5,142 +5,258 @@ import Image from 'next/image'
+                        </Box>
+                        <Box
+                            sx={{
+                                display: { xs: "flex", md: "none", lg: "none" },
+                                "& .MuiIconButton-root": {
+                                    color: "primary",
+                                },
+                            }}
+                        >
+                            <IconButton
+                                size="large"
+                                aria-label="website navigation"
+                                aria-controls="menu-appbar"
+                                aria-haspopup="true"
+                                onClick={handleOpenNavMenu}
+                            >
+                                <MenuIcon />
+                            </IconButton>
+                            <Menu
+                                id="menu-appbar"
+                                anchorEl={anchorElNav}
+                                anchorOrigin={{
+                                    vertical: "bottom",
+                                    horizontal: "left",
+                                }}
+                                keepMounted
+                                transformOrigin={{
+                                    vertical: "top",
+                                    horizontal: "left",
+                                }}
+                                open={Boolean(anchorElNav)}
+                                onClose={handleCloseNavMenu}
+                            >
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/stories">
+                                        <Typography textAlign="center" color="primary">
+                                            Stories
+                                        </Typography>
+                                    </Link>
+                                </MenuItem>
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/paintings/all">
+                                        <Typography textAlign="center" color="primary">
+                                            Paintings
+                                        </Typography>
+                                    </Link>
+                                </MenuItem>
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/manuscripts">
+                                        <Typography textAlign="center" color="primary">
+                                            Manuscripts
+                                        </Typography>
+                                    </Link>
+                                </MenuItem>
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/research/incipit-tool">
```

Comment:
The drop downs aren't working for mobile view, when you click research it just tries to redirect you to `/research/incipit-tool`